rq/cli/helpers.py

```diff
@@ -198,6 +202,82 @@ def setup_loghandlers_from_args(verbose, quiet, date_format, log_format):
     setup_loghandlers(level, date_format=date_format, log_format=log_format)
 
 
+def job_func(func, args, kwargs):
+    try:
+        importlib.import_module(func)  # execute file
+    except ImportError:
+        return import_attribute(func)(*args, **kwargs)
+
+
+def parse_function_arg(argument):
```

Comment:
We'll need a testcase for this function that thoroughly tests all possibilities so we know this is correct.